src/utils/buildArtifactUrl.js

```diff
@@ -0,0 +1,49 @@
+import { isNil } from 'ramda';
+import getIconFromMime from './getIconFromMime';
+import urls from './urls';
+
+export default ({ name, contentType, namespace, runId, taskId }) => {
```

Comment:
I'm not sure we need a lot of this because I built into taskcluster-web-server that you get the artifact URLs automatically built from the server instead of having to do this on the client.

Reply:
@eliperelman I didn't get a URL response from tc-web-server. The `url` field was `null`. Is this a bug in tc-web-server?

Reply:
Possibly, but it should be a property of the artifact itself. Like `artifact.url`. Maybe try it out with the GraphQL playground.